### PR TITLE
Create guide/overview for the SIG Security External Audit subproject

### PR DESCRIPTION
**Pull request by reylejano**
This PR creates an initial README.md for the SIG Security External Audit subproject.
The REAMD.md adds an overview and guide for the SIG Security External Audit subproject.

/sig security

### CODE REVIEW

File: sig-security/sig-security-external-audit/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## RFP
+
+The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the `kubernetes/community/sig-security` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
+
+Previous RFPs:
+  - [2019](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2019/RFP.md)
+  - [2021](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2021/RFP.md)
+
+As efforts begin for the year's security audit, create a tracking issue for the security audit in `kubernetes/community` with the `sig/security` label.
```
Comment (IanColdwater) [inline sig-security/sig-security-external-audit/README.md:23]: Would it make sense to add another label for this so you don't have to sift through all the other sig/security things to find the ones that pertain specifically to the audit?

Comment (reylejano) [reply sig-security/sig-security-external-audit/README.md:23]: @IanColdwater, good point, how about adding a `/label external-audit`

File: sig-security/sig-security-external-audit/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - [Request for Proposal (RFP)](#rfp)
+    - [Security Audit Scope](#security-audit-scope)
+    - [Vendor and Community Questions](#vendor-and-community-questions)
+  - [Review of Proposals](#review-of-proposals)
+  - [Vendor Selection](#vendor-selection)
+  - [Deliverables](#deliverables)
+
+## RFP
+
+The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the `kubernetes/community/sig-security` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
```
Comment (aasmall) [inline sig-security/sig-security-external-audit/README.md:17]: I don't think sig-security is its own repo. just a folder

```suggestion
The subproject produces a RFP for a third-party, comprehensive security audit. The subproject publishes the RFP in the 'sig-security' folder in the `kubernetes/community/` repository. The subproject defines the scope, schedule, methodology, selection criteria, and deliverables in the RFP.
```

File: sig-security/sig-security-external-audit/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  - Proposal reviewer is employed by a vendor who submitted a proposal
+  - Proposal reviewer has financial interest directly tied to the audit
+
+Should a conflict arise during the proposal review, reviewers should notify the subproject owner and SIG Security chairs when they become aware of the conflict.
+
+> The _Conflict of Interest_ section is inspired by the [CNCF Security TAG security reviewer process](https://github.com/cncf/tag-security/blob/main/assessments/guide/security-reviewer.md#conflict-of-interest).
+
+## Vendor Selection
+
+On the vendor selection date, the subproject will publish a the selected vendor in the `kubernetes/community/sig-security` repository. [Example from the 2019 audit](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2019/RFP_Decision.md).
```
Comment (aasmall) [inline sig-security/sig-security-external-audit/README.md:59]:
```suggestion
On the vendor selection date, the subproject will publish a the selected vendor in the 'sig-security' folder in the `kubernetes/community` repository. [Example from the 2019 audit](https://github.com/kubernetes/community/blob/master/sig-security/security-audit-2019/RFP_Decision.md).
```

File: sig-security/sig-security-external-audit/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## Deliverables
+
+The deliverables of the audit are defined in the RFP e.g. findings report, threat model, white paper, audited reference architecture spec (with yaml manifests) and published in the `kubernetes/community/sig-security` repository. [Example from the 2019 audit](https://github.com/kubernetes/community/tree/master/sig-security/security-audit-2019/findings)
```
Comment (aasmall) [inline sig-security/sig-security-external-audit/README.md:63]:
```suggestion
The deliverables of the audit are defined in the RFP e.g. findings report, threat model, white paper, audited reference architecture spec (with yaml manifests) and published in the 'sig-security' folder in the `kubernetes/community` repository. [Example from the 2019 audit](https://github.com/kubernetes/community/tree/master/sig-security/security-audit-2019/findings)
```

**Review by aasmall — APPROVED** [review]
Looks great, just a minor nit regarding referring to `sig-security` as a repo instead of a folder.

I would also suggest (non-blocking) that we wrap at a fixed column length, 80 or 120 are common. this improves raw readability and reduces merge conflicts. There are various IDE extensions to make this easy.

/LGTM

File: sig-security/sig-security-external-audit/README.md (hunk 1 above)
Comment (reylejano) [reply sig-security/sig-security-external-audit/README.md]: looks like `/label external-audit` is not allowed

File: sig-security/sig-security-external-audit/README.md (hunk 2 above)
Comment (reylejano) [reply sig-security/sig-security-external-audit/README.md]: applied suggestion in commit 947b473

File: sig-security/sig-security-external-audit/README.md (hunk 4 above)
Comment (reylejano) [reply sig-security/sig-security-external-audit/README.md]: applied suggestion in commit 947b473

File: sig-security/sig-security-external-audit/README.md (hunk 3 above)
Comment (reylejano) [reply sig-security/sig-security-external-audit/README.md]: applied suggestion in commit 947b473

**Comment by tabbysable** [comment]
Adding a separate label for issues related to the audit is still a good idea, but something that can be revisited later.

/label tide/merge-method-squash
/lgtm
/unhold

**Pull request merged**